extracted/konan.metadata/src/ext_options.proto1

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I believe this file is copied from `backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/`.
Is the original version still required to be in place?

Reply:
I am planning to drop all *.proto1 files in Kotlin/Native project (both that lay in `backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/` directory and those that were copied to `extracted/konan.metadata/src`) after transferring `extracted` to Big Kotlin repo.

Because all these files are actually copies of the corresponding *.proto files in Big Kotlin project. And I believe they should not remain in Kotlin/Native repo.